lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -454,11 +454,11 @@ public function getMigrations()
      */
     public function getVersion($version)
     {
-        if (!isset($this->migrations[$version])) {
-            throw MigrationException::unknownMigrationVersion($version);
+        if (isset($this->migrations[$version])) {
+            return $this->migrations[$version];
```

Comment:
Having the safeguard first is part of our coding standards. So such change is a no-go.